### PR TITLE
Update seiko.json

### PR DESCRIPTION
**Pull request by seikoodev**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview
https://seiko.is-a.dev (seiko-7oa.pages.dev)

### CODE REVIEW

**Comment by Delta-0mega** [comment]
Is it possible to add several records?
For domain & email routing

File: domains/seiko.json [hunk 1]
```diff
@@ -5,6 +5,12 @@
...
     },
     "record": {
-        "CNAME": "seiko-7oa.pages.dev"
+        "CNAME": "seiko-7oa.pages.dev",
+        "MX": [
+            "route1.mx.cloudflare.net.",
+            "route2.mx.cloudflare.net.",
+            "route3.mx.cloudflare.net."
+        ],
+        "TXT": "v=spf1 include:_spf.mx.cloudflare.net ~all"
```
Comment (CuteDog5695) [inline domains/seiko.json:14]: Can't use CNAME with any other records.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/seiko.json (hunk 1 above)
Comment (Delta-0mega) [reply domains/seiko.json:14]: Okay tysm

Comment (Delta-0mega) [reply domains/seiko.json:14]: So, Is it possible to use cloudflare page without CNAME ?

**Pull request closed without merging**